packages/icons/src/img/sprite.svg

```diff
@@ -497,6 +500,10 @@
         d="M3.333 3.25a.083.083 0 0 0-.083.083v8.417h13.5V3.333a.083.083 0 0 0-.083-.083H3.333Zm14.917.083a1.583 1.583 0 0 0-1.583-1.583H3.333A1.583 1.583 0 0 0 1.75 3.333v10a1.583 1.583 0 0 0 1.583 1.583h4.15l-.228 1.834h-.588a.75.75 0 0 0 0 1.5h6.666a.75.75 0 0 0 0-1.5h-.588l-.229-1.834h4.15a1.584 1.584 0 0 0 1.584-1.583v-10Z"
         clip-rule="evenodd"
       />
+    <symbol viewBox="0 0 20 20" id="lp-icon-chart-forest">
+      <path fill-rule="evenodd" clip-rule="evenodd" d="M8.37803 3.25C8.68691 2.37611 9.52034 1.75 10.5 1.75H12.5C13.4797 1.75 14.3131 2.37611 14.622 3.25H18C18.4142 3.25 18.75 3.58579 18.75 4C18.75 4.41421 18.4142 4.75 18 4.75H14.622C14.3131 5.62389 13.4797 6.25 12.5 6.25H10.5C9.52034 6.25 8.68691 5.62389 8.37803 4.75H5C4.58579 4.75 4.25 4.41421 4.25 4C4.25 3.58579 4.58579 3.25 5 3.25H8.37803ZM10.5 3.25C10.0858 3.25 9.75 3.58579 9.75 4C9.75 4.41421 10.0858 4.75 10.5 4.75H12.5C12.9142 4.75 13.25 4.41421 13.25 4C13.25 3.58579 12.9142 3.25 12.5 3.25H10.5Z" fill="#3F454C"/>
```

Comment:
@matthewferry we don't want fills set right?

Reply:
ah yep, good catch!